src/index.js

```diff
@@ -0,0 +1,84 @@
+const crypto = require('crypto');
+const assert = require('assert');
+
+const key = process.env.HASHING_KEY;
```

Comment:
since this is a library this should be an input param and thus there should be no need for .env files